samples/test_all_apis.py

```diff
@@ -581,23 +582,26 @@ def get_saw_event():
             return GPIO.event_detected(pin_data['in_a'])
 
     if specify_callback:
-        args = {'callback': callback}
+        args = {'callback': callback, 'polltime': 0.2}
     else:
-        args = {}
+        args = {'polltime': 0.2}
+
+    time.sleep(0.2)
+    # By default, the poll time is 0.2 seconds, too
```

Comment:
Can you comment here why the sleep time is 0.2 here.

Reply:
Got it.